glean-core/src/metrics/string.rs

```diff
@@ -11,6 +11,9 @@ use crate::Glean;
 
 const MAX_LENGTH_VALUE: usize = 100;
 
+/// # A string metric
+///
+/// Record an Unicode string value with arbitrary content.
```

Comment:
Should we mention that it's length limited and point to the `MAX_LENGTH_VALUE` above?